components/scream/src/physics/p3/p3_f90.cpp

```diff
@@ -35,7 +35,7 @@ FortranData::FortranData (Int ncol_, Int nlev_)
   do_predict_nc = true;
   do_prescribed_CCN = true;
   dt = -1; // model time step, s; set to invalid -1
-  it = 1;  // seems essentially unused
+  it = 1;
```

Comment:
Is it used then? It seems to me it's used in `p3_run_and_cmp`, but not in here (which would make sense).

Reply:
It is. It's used when the Data is passed to p3_main.